docs/usage/cli.rst

```diff
@@ -117,7 +117,24 @@ Options
 +-------------------------------------+---------------------------+-----------------------------------------------------------------+
 | ``--debug``                         | ``LITESTAR_DEBUG``        | Run the application in debug mode                               |
 +-------------------------------------+---------------------------+-----------------------------------------------------------------+
+| ``--reload-dir``                    | ``LITESTAR_RELOAD_DIRS``  | Specify directories to watch for reload.                        |
```

Comment:
How does the user specify multiple reload directories with the `LITESTAR_RELOAD_DIRS` variable?